File: Mahasiswa.java
public class Mahasiswa extends SivitasAkademik
{
	private String nim;
	private String prodi;
	private String fakultas;
	
	Mahasiswa(){
	}
	public void setNim(String nim){
		this.nim = nim;
	}
	public String getNim(){
		return this.nim;
	}
	public void setProdi(String prodi){
		this.prodi = prodi;
	}
	public String getProdi(){
		return this.prodi;
	}
	public void setFakultas(String fakultas){
		this.fakultas = fakultas;
	}
	public String getFakultas(){
		return this.fakultas;
	}
}

File: php/Mahasiswa.php
<?php

//Mahasiswa adalah bagian dari sivitas akademik

class Mahasiswa extends SivitasAkademik
{
	private $nim;
	private $prodi;
	private $fakultas;
	
	public function __construct(){
		parent::__construct();
		$this->nim = "-";
		$this->prodi = "-";
		$this->fakultas = "-";
	}
	public function setNim($nim){
		$this->nim = $nim;
	}
	public function getNim(){
		return $this->nim;
	}
	public function setProdi($prodi){
		$this->prodi = $prodi;
	}
	public function getProdi(){
		return $this->prodi;
	}
	public function setFakultas($fakultas){
		$this->fakultas = $fakultas;
	}
	public function getFakultas(){
		return $this->fakultas;
	}
}

?>